### PR TITLE
fix: New UI menu bar issues

### PR DESCRIPTION
**Pull request by JeffreyMa597**
What problem does this PR solve?
------
As title.

Ref:https://github.com/Magickbase/neuron-public-issues/issues/158

Check List
------

### Test
e2e Test

### CODE REVIEW

File: packages/neuron-ui/src/styles/mixin.scss [hunk 1]
```diff
@@ -175,8 +175,12 @@
...
       g,
       path {
-        fill: $hoverColor
+        fill: $hoverColor;
       }
     }
   }
 }
+
+@mixin cursor-mode {
```
Comment (yanguoyu) [inline packages/neuron-ui/src/styles/mixin.scss:184]: Is it necessary to define one property with `mixin`? I think `cursor: pointer` is clear to replace `@include cursor-mode;`

**Review by yanguoyu — APPROVED** [review]

**Comment by WhiteMinds** [comment]
Scroll bars and unexpected overflows when shrinking windows

image

The language option is missing in the settings, I can't switch languages.

**Review by WhiteMinds — CHANGES_REQUESTED** [review]
Scroll bars and unexpected overflows when shrinking windows

image

The language option is missing in the settings, I can't switch languages.

**Comment by JeffreyMa597** [comment]
> Scroll bars and unexpected overflows when shrinking windows

updated. 
image


> The language option is missing in the settings, I can't switch languages.

You can select them over here.
image

**Review by WhiteMinds — APPROVED** [review]

**Comment by WhiteMinds** [comment]
> You can select them over here.

I can't believe I looked at it 3 times and didn't notice it 🫠. Is it my problem or a UI issue?

**Comment by Keith-CY** [comment]
> > You can select them over here.
> 
> I can't believe I looked at it 3 times and didn't notice it 🫠. Is it my problem or a UI issue?

@Danie0918 @Sven-TBD please check if it is user friendly

**Comment by Danie0918** [comment]
> > > You can select them over here.
> > 
> > 
> > I can't believe I looked at it 3 times and didn't notice it 🫠. Is it my problem or a UI issue?
> 
> @Danie0918 @Sven-TBD please check if it is user friendly

Language setting is not a high-frequency operation, so it is reasonable to put it in the settings and continue the previous user habits. The problem here is that the language of the current settings is weakened, so users can't find it quickly, and this needs to be optimized by adding an icon or deepening the color.  @Kirl

**Comment by Kirl70** [comment]
> > > > You can select them over here.
> > > 
> > > 
> > > I can't believe I looked at it 3 times and didn't notice it 🫠. Is it my problem or a UI issue?
> > 
> > 
> > @Danie0918 @Sven-TBD please check if it is user friendly
> 
> Language setting is not a high-frequency operation, so it is reasonable to put it in the settings and continue the previous user habits. The problem here is that the language of the current settings is weakened, so users can't find it quickly, and this needs to be optimized by adding an icon or deepening the color. @Kirl70

Icons have been added in front of the options as an indication:
https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?type=design&node-id=1165%3A12477&t=JhtdSRXHKrCkhAGF-1
image

**Comment by JeffreyMa597** [comment]
> Icons have been added in front of the options as an indication:
https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?type=design&node-id=1165%3A12477&t=JhtdSRXHKrCkhAGF-1

Updated.

**Review by FrederLu — APPROVED** [review]

**Review by devchenyan — APPROVED** [review]

**Pull request merged**